src/main/webapp/index.html

```diff
@@ -22,71 +22,33 @@
       <button onclick="getClassID()">Get class ID</button>
       <br /><br />
 
-      <textarea name="class-input">Write opinion on class here.</textarea>
+      <textarea name="class-input" maxlength="1500">Write opinion on class here.</textarea>
```

Comment:
This is good but keep in mind that users *can* get around this, you'll still need to handle a longer maxlength on the server. (Something to keep in mind, no change required just now.)